workflow-templates/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: Link Markdown
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check_links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Actions Checkout
+        uses: actions/checkout@v2
+      - name: Check Links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
This only covers link checks for Markdown. Could you also please add linting?

Also, please add filters so that it's only executed for Markdown files, rather than for all files. Check out whether either categories or file pattern as a parameter can help: https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template

Reply:
Ok yes!

Reply:
lint code were upload

Reply:
I set a filter by ".md" files so it wouldnt check all files.